src/main/java/org/folio/edge/patron/model/Hold.java

```diff
@@ -239,7 +190,11 @@ public boolean equals(Object obj) {
     } else if (!expirationDate.equals(other.expirationDate)) {
       return false;
     }
-    if (fulfillmentPreference != other.fulfillmentPreference) {
+    if (pickupLocationId == null) {
+      if (other.pickupLocationId != null) {
+        return false;
+      }
+    } else if (!pickupLocationId.equals(other.pickupLocationId)) {
```

Comment:
Are there any tests for this new code? It may be academic, but I think this method was created for a reason, so we may need to have a test for this if there isn't one already.

Reply:
Yes. There is an existing `testEqualsContract` in the model tests:
https://github.com/folio-org/edge-patron/blob/f1c95865c2cebf37bd4d7ca31a84537d711ca072/src/test/java/org/folio/edge/patron/model/HoldTest.java#L96-L99